src/common/_modules/application_gateway/main.tf

```
@@ -412,18 +412,35 @@ module "app_gw" {
       }
     }
 
-    ipatente-io-pagopa-it = {
+    vehicles-ipatente-io-pagopa-it = {
       protocol           = "Https"
-      host               = format("ipatente.%s", var.public_dns_zones.io.name)
+      host               = format("vehicles.%s", var.public_dns_zones.ipatente_io_pagopa_it.name)
       port               = 443
       ssl_profile_name   = format("%s-ssl-profile", var.project)
       firewall_policy_id = null
 
       certificate = {
-        name = var.certificates.ipatente_io_pagopa_it
+        name = var.certificates.vehicles_ipatente_io_pagopa_it
         id = replace(
-          data.azurerm_key_vault_certificate.app_gw_ipatente_io.secret_id,
-          "/${data.azurerm_key_vault_certificate.app_gw_ipatente_io.version}",
+          data.azurerm_key_vault_certificate.app_gw_vehicles_ipatente_io.secret_id,
+          "/${data.azurerm_key_vault_certificate.app_gw_vehicles_ipatente_io.version}",
+          ""
+        )
+      }
+    }
+
+    licences-ipatente-io-pagopa-it = {
+      protocol           = "Https"
+      host               = format("licences.%s", var.public_dns_zones.ipatente_io_pagopa_it.name)
+      port               = 443
+      ssl_profile_name   = format("%s-ssl-profile", var.project)
+      firewall_policy_id = null
+
+      certificate = {
+        name = var.certificates.licences_ipatente_io_pagopa_it
+        id = replace(
+          data.azurerm_key_vault_certificate.app_gw_licences_ipatente_io.secret_id,
+          "/${data.azurerm_key_vault_certificate.app_gw_licences_ipatente_io.version}",
           ""
         )
```

Should make the code easier with the same result
```suggestion
        id = data.azurerm_key_vault_certificate.app_gw_licences_ipatente_io.versionless_secret_id
```